tests/e2e/editor-selector.spec.js

```diff
@@ -0,0 +1,38 @@
+const {test, expect} = require('@playwright/test');
+import {login} from './tools/lib/login';
+
+test('Test the creation of a new Post', async ({page, context}) => {
+  test.setTimeout(240 * 1000);
+  await page.goto('./');
+  await login(page, context);
+
+  // Creating and navigating to new post
+  await page.goto('./wp-admin/post-new.php');
+
+  // On dev instance, need to close modal so test can continue
+  const closeButton = page.locator('.components-modal__header button');
+  if (await closeButton.isVisible()) {
+    await closeButton.click();
+  }
+
+  await page.locator('.editor-post-title__input').click();
+  await page.locator('h1.editor-post-title').fill('Test Post');
+  await page.locator('.block-editor-block-list__layout').click();
+  await page.locator('p.is-selected.wp-block-paragraph').fill('This is a test Post.');
+  await page.keyboard.press('Enter');
+  await page.locator('p.is-selected.wp-block-paragraph').fill('/youtube');
+  await page.keyboard.press('Enter');
+  await page.locator('[aria-label="YouTube URL"]').fill('https://youtu.be/3gPvDDHU41E');
+  await page.getByRole('button', {name: 'Embed'}).click();
+  await page.getByRole('button', {name: 'Publish', exact: true}).click();
```

Comment:
Is there a way to select the buttons without using the name? 🤔 Because in test instances sometimes we use a different language in the backend, in which case these might fail